src/include/sof/audio/volume.h

```diff
@@ -110,6 +110,8 @@ struct comp_data {
 	int32_t vol_min;			/**< minimum volume */
 	int32_t vol_max;			/**< maximum volume */
 	int32_t	vol_ramp_range;			/**< max ramp transition */
+	bool muted[SOF_IPC_MAX_CHANNELS];	/**< set if channel in muted */
```

Comment:
@singalsu typo here "set if channel is muted"?

Reply:
Oops, fixed it!